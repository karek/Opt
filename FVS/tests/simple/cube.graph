a b
b c
c d
a d
x y
y z
z t
t x
a x
b y
c z
d t
