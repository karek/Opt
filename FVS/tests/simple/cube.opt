3
a c y 
