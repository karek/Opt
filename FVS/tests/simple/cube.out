3
d y t 
